assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -346,76 +349,132 @@ export default function EffectChooser({
               <ContentWrapper>{__('Zoom', 'web-stories')}</ContentWrapper>
               <ZoomOutAnimation>{__('Zoom', 'web-stories')}</ZoomOutAnimation>
             </GridItemFullRow>
-            <GridItem
-              aria-label={__('Pan Left Effect', 'web-stories')}
-              onClick={(event) =>
-                handleOnSelect(event, PAN_MAPPING[DIRECTION.LEFT_TO_RIGHT], {
-                  animation: BACKGROUND_ANIMATION_EFFECTS.PAN.value,
-                  panDir: DIRECTION.LEFT_TO_RIGHT,
-                })
+            <WithTooltip
+              title={
+                disabledBackgroundEffects.includes(
+                  PAN_MAPPING[DIRECTION.LEFT_TO_RIGHT]
+                )
+                  ? disabledTypeOptionsMap[
+                      BACKGROUND_ANIMATION_EFFECTS.PAN.value
+                    ]?.tooltip
+                  : ''
               }
```

Comment:
This appears to have the unfortunate side effect of killing the keyboard functionality because the tooltip's wrapping the button and the tooltip div isn't focusable. 
![Screen Shot 2020-12-08 at 1 30 26 PM](https://user-images.githubusercontent.com/10720454/101537985-a5b66a00-3959-11eb-8206-681e5e805674.png)

Reply:
Ahh great catch! will just move the tooltip wrapper inside the button as a quickfix for now